src/sentry/event_manager.py

```diff
@@ -851,6 +852,8 @@ def save(self, project, raw=False):
         if is_regression and not raw:
             regression_signal.send_robust(sender=Group, instance=group)
 
+        metrics.timing('event.latency', received_timestamp - recorded_timestamp)
```

Comment:
I'd like to tag this with project, please let me know if a.) that's viable, b.) if it is, what tag key I should be using here

Reply:
@JTCunning to confirm, but `project_id` I'm pretty sure is fine.

Reply:
`events.latency`, `project_id` is also fine.

Reply:
Updated this to pluralize (`event` to `events`) and added the `project_id` tag with 8693eb98c3cec9e3b62ec1af45fc09bb1895e026